builder/builder_test.go

```diff
@@ -264,3 +290,23 @@ func TestRollback(t *testing.T) {
 	}
 	// We trust that the other parts of a tx store rollback were done as well.
 }
+
+// TODO: should this be in a helper file?
```

Comment:
Personally, I think it's fine where it is. I usually only add a `helpers.go` file when I want to keep code separate from copied/generated code, or can't think of a better name.